net/unbound/files/unbound.hotplug

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+[ "$ACTION" = ifup ] || exit 0
+/etc/init.d/unbound enabled || exit 0
+/etc/init.d/unbound restart
```

Comment:
This seems like a reasonable thing to do. Please re-submit as a separate, small change.